src/shared_modules/utils/flatbuffers/schemas/syscheck_deltas.fbs

```diff
@@ -14,7 +14,7 @@ table Attributes {
     user_name: string;
     gid: string;
     group_name: string;
-    inode: long;
```

Comment:
I think this is unnecessary, flatbuffer interprets numeric strings as a long.

https://github.com/google/flatbuffers/blob/c7b9dc83f58fe72162b076ba9c88d682f5d0eda6/src/idl_parser.cpp#L2166-L2194

Reply:
Thanks for the suggestion, I revert the string type to long